services/services.rst

```diff
@@ -171,15 +99,17 @@ For accessing on-line, you will need to `install clients <https://zoom.us/downlo
 For scheduled list of upcoming teleconferences for PFS technical team, 
 the project is providing a list of scheduled teleconferences calendar at 
 `google calendar <https://calendar.google.com/calendar/embed?src=su0pbsaull17etlj62tet5anm0%40group.calendar.google.com>`_.
```

Comment:
For upcoming teleconference of PFS technical team, the project is providing the meeting schedule at ...

(Trying to make sentence shorter)